Cython/Compiler/Symtab.py

```diff
@@ -428,6 +425,26 @@ def mangle_class_private_name(self, name):
         if self.parent_scope:
             return self.parent_scope.mangle_class_private_name(name)
         return name
+    
+    def nomangle(self, name):
+        r"""
+        Allows unicode names to be translated to valid C/C++ code
```

Comment:
Honestly, I would leave this out for now, or at least move it to a separate PR. It seems to deserve a separate discussion.